### ISSUE TITLE
DEPRECATED Action signature.  Must be `cli.ActionFunc`

### ISSUE THREAD

**Issue by mblakele** [nadnerb/terraform_exec#11]
I see this on every execution of `terraform_exec`:

```
DEPRECATED Action signature.  Must be `cli.ActionFunc`.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature
```

**Comment by nadnerb** [comment]
Yeah this is due to changes done to the cli tool used. I will have have to upgrade that dependency.

**Comment by nadnerb** [comment]
this should be fixed by `34c89ca`
